SocialFilter/sections/css_extended.txt

```diff
@@ -833,7 +832,7 @@ and-engineer.com#?#div[class^="css-"]:has(> div[class] > a[href^="https://twitte
 and-engineer.com#?##__next div[class^="css-"]:has(> div[class] + div[class^="css-"] > div.chakra-stack > div[class^="css-"] > a[href^="https://twitter.com/share?url"])
 ! https://github.com/AdguardTeam/AdguardFilters/issues/102614#issuecomment-1056823395
 youtube.com#?#yt-third-party-share-target-section-renderer > .yt-third-party-share-target-section-renderer ~ yt-icon-button#scroll-button-forward
-youtube.com#?#yt-share-target-renderer > button[title] > div.style-scope:contains(/WhatsApp|Facebook|Twitter|카카오스토리|Reddit|ВКонтакте|Одноклассники|ОК|Pinterest|Blogger|Tumblr|LinkedIn|Skyrock|Mix|goo|KakaoTalk/):upward(button[title])
+youtube.com#?#yt-share-target-renderer > button[title] > div.style-scope:contains(/WhatsApp|Facebook|Twitter|카카오스토리|Reddit|ВКонтакте|Одноклассники|ОК|OK|VK|Pinterest|Blogger|Tumblr|LinkedIn|Skyrock|Mix|goo|KakaoTalk/):upward(button[title])
```

Comment:
Two times OK?

Reply:
![ad](https://user-images.githubusercontent.com/110785003/183310133-71a84b83-446d-43a2-a66b-fd570fffa3e3.png)

Reply:
Thank you, on mobile it looked the same.